### DISCUSSION TITLE
Food Storage / Battery Tracker

### DISCUSSION THREAD

**Discussion by ceonelson** [lrnselfreliance/wrolpi#268] [Ideas]
Just found your project from DH and am excited to give it a try soon!

If you aren't familiar with https://github.com/grocy/grocy it could be a solution to the food storage tracker aspect :)

It also has a battery (and chore) tracker built in too which could be helpful both before and during WRoL to know which batteries have been recharged the most, track last charge time, what they're for, next recharge time, etc.

One could also add equipment and store their manuals and notes using grocy as well

**Comment by lrnselfreliance** [comment]
Thanks for the info. I will look into this and see if it can work with WROLPi.
